ios/Divoc/Imported/Networking/TracingLocalPush.swift

```diff
@@ -102,6 +102,16 @@ class TracingLocalPush: NSObject {
     center.add(request, withCompletionHandler: completionHandler)
   }
 
+  private let notificationIdentifierInfoBox = "stayaway.notification.infobox"
+  func scheduleInfoBoxNotification(title: String, body: String, url: String, completionHandler: ((Error?) -> Void)?) {
```

Comment:
should the completionHandler have a default value of nil ? since in the Manager you're passing nil.. 🤔

Reply:
ok